giga_connectome/utils.py

```diff
@@ -199,3 +207,60 @@ def check_path(path: Path, verbose=True):
         if verbose:
             print(f"Specified path already exists, using {path} instead.")
     return path
+
+
+def create_ds_description(output_dir: Path) -> None:
+    """Create a dataset_description.json file."""
+    ds_desc: dict[str, Any] = {
+        "BIDSVersion": "1.9.0",
+        "License": None,
+        "Name": None,
+        "ReferencesAndLinks": [],
+        "Authors": [
+            "Foo",
+            "Bar",
+        ],
+        "DatasetDOI": None,
+        "DatasetType": "derivative",
+        "GeneratedBy": [
+            {
+                "Name": "giga_connectome",
+                "Version": __version__,
+                "CodeURL": "https://github.com/SIMEXP/giga_connectome.git",
+            }
+        ],
+        "HowToAcknowledge": (
+            "Please refer to our repository: "
+            "https://github.com/SIMEXP/giga_connectome.git."
+        ),
+    }
+    with open(output_dir / "dataset_description.json", "w") as f:
+        json.dump(ds_desc, f, indent=4)
+
+
+def create_sidecar(output_path: Path) -> None:
+    """Create a JSON sidecar for the connectivity data."""
+    metadata: dict[str, Any] = {
+        "NodeFiles": "REQUIRED",
+        "Measure": "Pearson correlation",
+        "MeasureDescription": "Pearson correlation",
+        "Weighted": "REQUIRED",
```

Comment:
should be updated

Reply:
@htwangtw the measures we get is not weighted in any way, right?

Reply:
No not weighted